### DISCUSSION TITLE
Сolor change

### DISCUSSION THREAD

**Discussion by ghost** [Hanziness/AnotherPomodoro#179] [Ideas & feature requests]
Add color change for text and background.

**Comment by Hanziness** [comment]
Theming was partially implemented in the app. There are settings values that control the colours of the work, pause and long pause sections, but I did not want to expose them because it's easy to accidentally break the contrast through full customization and I imagined I'd have to handle multiple edge cases (eg. auto pick a matching font colour for the background).

I'll try to experiment with it. Which option would you consider better?

1. Full hex colour picker for all three sections' colours, or
2. prebuilt themes that change all three colours together?

**Reply by ghost** [reply]
I understand why this is so.

1. It is better for people like me with visual impairments.
2. For most people, although the default theme of the tomato is so interesting.

**Reply by Hanziness** [reply]
Hello! It's been quite some time, but section colours can now be customized to some extent. There are 5 inbuilt themes, at least one of them is verified to be colourblind-safe and the user can also set their own colours (with some limitations regarding saturation and lightness). This is implemented in #341 and is now in the `develop` branch, the feature will be released in version 1.5 :)

Sorry it took so long!
